### DISCUSSION TITLE
A little confusion of the 'Parent Child Variants' document

### DISCUSSION THREAD

**Discussion by louis96108** [mtex-toolbox/mtex#944] [Ask Anything] [answered]
Hi everyone,
In the Document>Parent Child Variants, pole figure is plotted to prove the Burger's Orientation Relationship like the picture below.
(https://user-images.githubusercontent.com/69557894/114190427-e1038c80-997d-11eb-9559-62b18e985586.png)
Follow the code in the document，the second pole figure,(111)(11-20), would be  'Plane' Pole Figure, but I think it should be 'Direction' Pole figure，[111][11-20]. Could anyone tell me what it should be and why? Thanks a lot!

Sincerely,
Louis

**Comment by louis96108** [comment, marked as answer]
Hi, everyone
Two days passed and I still have doubts about this problem. Looking forward to your answer. Wish you a good day!

Sincerely,
Louis

**Reply by ralfHielscher** [reply]
Hi Louis,

thank you for pointing this out. It will be corrected soon. 

Ralf.

**Reply by louis96108** [reply]
Hi Ralf,
Thanks for your reply. I will continue to use MTEX in the future and give you feedback.

Cheers,
Louis
